### PR TITLE
Readme km

### PR DESCRIPTION
**Pull request by kmcgrevey**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling - no new features
### Why is this change required? What problem does it solve?
This creates a polished `README.md` file to summarize and document this backend portion of the project.
### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
Imbedding the `Swagger` API window for live interaction within the markdown file proved difficult.  Solution was to provide a screen grab of the endpoints with a corresponding link that directs to the interactive Swagger diagrams.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+- To run server on `localhost:5000/`:
+  ```
+  python run.py
+  ```
+
+## Testing and Coverage
+- To run a testing and coverage report utilizing the included `coverage` module:
+  ```
+  nosetests --cover-package=application --with-coverage
```
Comment (StephanieFriend) [inline README.md:76]: I love this setup instructions! Very clear and precise!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ - Stephanie Friend ([GitHub](https://github.com/StephanieFriend), [LinkedIn](https://www.linkedin.com/in/s-friend/))
+ - Kevin McGrevey ([GitHub](https://github.com/kmcgrevey), [LinkedIn](www.linkedin.com/in/‎kevin-mcgrevey‎-8660958/))
+ - Andrew Reid ([GitHub](https://github.com/reid-andrew), [LinkedIn](https://www.linkedin.com/in/reida/))
+
+ ## Technologies and Frameworks
+
+- Back-End
+  - Language: Python 3.8.4
+  - Framework: Flask 1.1.2
+  - Testing: Nose
```
Comment (StephanieFriend) [inline README.md:90]: Great for adding the versions!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Auctionable Change API [![Build Status](https://travis-ci.com/Auctionable-Change/auctionable_change_api.svg?branch=master)](https://travis-ci.com/Auctionable-Change/auctionable_change_api)
+
+## Description
+
+**Auctionable Change API** is the backend API for our [Turing School of Software and Design](https://turing.io/) capstone project, **Auctionable Change**.
+
+The idea behind **A.change** is creating a platform in which an individual can post an item for sale/auction, set a minimum asking price and then select a charity to which the funds from that item's sale would be donated.
+
```
Comment (StephanieFriend) [inline README.md:8]: I like this description! Nice and concise. Easy to understand. Maybe if we have time further on we can elaborate on the buyer side too?

**Review by StephanieFriend — APPROVED** [review]
Great job! I made a few comments - most were praises of course! But I made one comment at the top that I thought we can work on if we have time in the future. Otherwise I think this is fantastic and thank you for taking this on!

**Review by reid-andrew — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (kmcgrevey) [reply README.md:8]: Yeah -- with the uploading receipt routine.  The FE might get that covered with their project description.

**Pull request merged**